10LessonTen/Checkpoint2/index.html

```diff
@@ -71,7 +71,77 @@
             <li>Disclaimer: A general statement that expresses the legal limits to the scope of the app.</li>
           </ol>
         </p>
-
+        <h1>Areas of Practice</h1>
+        <TABLE>
+           <TR>
+              <TD>Criminal Defense<p>Criminal Defense App also includes:
+I’m Being Arrested: At the click of a button send GPS location and a pre-set text message to a loved-one informing them of your client's arrest.
+​
+Call A Taxi: A quick way to get a taxi.
+​
+Criminal Defense form: A detailed form to send info directly to your law firm in preparation for their case.
+​
+Record an Audio Clip:  A quick audio recording method.
+​
+B.A.C. Calculator:  A blood alcohol calculator that provides a general estimate of alcohol level.</p></TD>
+              <TD>Personal Injury<p>Personal Injury App also includes:
+Call A Taxi: A quick way to get a taxi.
+​
+Capture Accident Data: A detailed form to send info directly to your law firm in preparation for the case.
+​
+Expense Log:  A method to log your expenses and send them to your law firm.
+​
+Flashlight: A quick source of light
+​
+Proof of Insurance: A back-up digital image of your insurance.</p></TD>
+           </TR>
+           <TR>
+             <TD>Consumer Law<p>Consumer Law App also includes:
+​
+Capture/Send Evidence: A detailed form to send info such as harassment calls from debt collectors directly to your law firm in preparation for the case.
+​
+Do Not Call Registry: A quick way to be added to the do not call registry.
+​
+Call/SMS Tracker: A way to track calls from debt collectors.
+​
+Household Debt Tracker: A database to track household debt.</p></TD>
+             <TD>Nursing Home Abuse<p>Nursing Home Abuse App also includes:
+​
+Call 911: A quick way to contact 911.
+​
+Help Center: A detailed form to send info directly to your law firm in preparation for the case.
+​
+Expense Log:  A method to log medical expenses for the case.
+​
+Flashlight: A quick source of light.
+​
+Proof of Insurance: A back-up digital image of health insurance.</p></TD>
+           </TR>
+           <TR>
+              <TD>Medical Malpractice<p>Medical Malpractice App also includes:
+​
+Call 911: A quick way to contact 911.
+​
+Injury Help Center: A detailed form to send info directly to your law firm in preparation for the case.
+​
+Expense Log:  A method to log medical expenses for the case.
+​
+Flashlight: A quick source of light.
+​
+Proof of Insurance: A back-up digital image of your insurance.</p></TD>
+              <TD>Workers' Comp<p>Workers Comp App also includes:
+​
+Call 911: A quick way to contact 911.
+​
+Capture Workers Comp info: A detailed form to send info directly to your law firm in preparation for the case.
+​
+Expense Log:  A method to log medical expenses for the case.
+​
+Flashlight: A quick source of light.
+​
+Proof of Insurance: A back-up digital image of your insurance.</p></TD>
+           </TR>
+        </TABLE>
```

Comment:
Great job Khadijah, you have really improved over the last few weeks! The website looks great! I would say you are steadily improving on your code readability. The code at the very end is a bit hard to read (I would say this starts in the table area). Keep up the good work.